### PR TITLE
fix: add SubhadeepZilong

### LINKED ISSUE #1132: Subhadeep Chakraborty
## Profile Screenshot

screencapture-github-SubhadeepZilong-2022-05-27-21_50_39

### PR DESCRIPTION
**Pull request by SubhadeepZilong**
resolves #1132

## Checklist for profile addition

- [x] Name the PR `fix: add <your-username>`.
- [x] You've added file named `your-username.json` in the [profiles](https://github.com/EddieHubCommunity/awesome-github-profiles/tree/main/profiles) directory.
- [x] You've linked the PR to the correct issue.

---

## Other additions/changes

- [x] Changes made in this PR are mentioned below
- [x] Search previous suggestions before making a new one, as yours may be a duplicate.
- [x] I acknowledge that all my contributions will be made under the project's license.
- [x] I've read the [contributing guidelines](https://github.com/EddieHubCommunity/awesome-github-profiles/blob/main/CONTRIBUTING.md)

## This PR adds/fixes

- SubhadeepZilong.json file has been added

### CODE REVIEW

File: profiles/SubhadeepZilong.json [hunk 1]
```diff
@@ -0,0 +1,6 @@
+{
+    "image": "https://user-images.githubusercontent.com/63146468/170739449-df9a415d-7b8a-41cc-9891-5ac2d26cce86.png",
+    "issueId": 1132,
+    "name": "Subhadeep Chakraborty",
+    "username": "SubhadeepZilong"
+  }
```
Comment (Panquesito7) [inline profiles/SubhadeepZilong.json:6]: Missing an endline. Without this endline, there could be issues with some IDEs and configurations.
```suggestion
}

```

**Review by Panquesito7 — CHANGES_REQUESTED** [review]

File: profiles/SubhadeepZilong.json [hunk 2]
```diff
@@ -0,0 +1,6 @@
+{
+    "image": "https://user-images.githubusercontent.com/63146468/170739449-df9a415d-7b8a-41cc-9891-5ac2d26cce86.png",
+    "issueId": 1132,
+    "name": "Subhadeep Chakraborty",
+    "username": "SubhadeepZilong"
```
Comment (Panquesito7) [inline profiles/SubhadeepZilong.json:5]: I think the indentation space should be 2 in here.
```suggestion
  "image": "https://user-images.githubusercontent.com/63146468/170739449-df9a415d-7b8a-41cc-9891-5ac2d26cce86.png",
  "issueId": 1132,
  "name": "Subhadeep Chakraborty",
  "username": "SubhadeepZilong"
```

**Review by schmelto — APPROVED** [review]
__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :ok_hand: This code looks good

**Review by Pradumnasaraf — APPROVED** [review]
Looks good to me, thank you for your contribution

__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :ok_hand: This code looks good&nbsp; | &nbsp;**Tags:** Efficient

**Pull request merged**